src/views/PermissionSend.vue

```diff
@@ -271,7 +289,12 @@ export default {
     },
     applyCustomFee({ fee }) {
       const presetFee = Object.entries(this.assetFees).find(
-        ([speed, speedFee]) => speed !== 'custom' && speedFee.fee === fee
+        ([speed, speedFee]) =>
+          speed !== 'custom' &&
```

Comment:
These few lines are really hard to follow. anyway to improve it?

you can name statements for example

const feeMatches = speedFee.fee.maxPriorityFeePerGas === fee.maxPriorityFeePerGas

Reply:
Fixed, splited to 2 varialbes: `isEIP1559Fee` and `isLegacyFee` for better readability